db/migrations/20241221234500_fix_types.go

```diff
@@ -0,0 +1,27 @@
+package migrations
+
+import (
+	"github.com/go-gormigrate/gormigrate/v2"
+	"gorm.io/gorm"
+)
+
+var _20241221234500_fix_types = &gormigrate.Migration{
+	ID: "20241221234500_fix_types",
+	Migrate: func(tx *gorm.DB) error {
+		if tx.Dialector.Name() == "postgres" {
+			err := tx.Exec(`
+CREATE SEQUENCE user_configs_id_seq;
```

Comment:
add comment: `// due to not explicitly setting the user_configs table to use auto increment in previous migrations`